keymaps/find-and-replace.cson

```diff
@@ -3,6 +3,9 @@
   'cmd-f': 'find-and-replace:show'
   'cmd-alt-f': 'find-and-replace:show-replace'
 
+'atom-text-editor':
+  'alt-a': 'project-find:show-in-parent-directory'
```

Comment:
I'm not finding any other editors using `alt-a`, I just can't find other free keybinding of two keys which can be handled by one hand. 

I try to use `alt-f` at first, but it is used as 'editor:move-to-end-of-word' by atom core.